spec/features/edit_vaccination_record_spec.rb

```diff
@@ -234,12 +215,24 @@
   end
 
   def given_i_am_signed_in
-    @organisation = create(:organisation, :with_one_nurse, ods_code: "R1L")
+    @organisation =
+      create(
+        :organisation,
+        :with_one_nurse,
+        :with_generic_clinic,
```

Comment:
Curious why this was needed? If you remember offhand.

Reply:
Turns out it wasn't needed, not sure why I put it in. I've reverted the change and amended the commit.